README.md

```diff
@@ -237,7 +242,6 @@ The `revoke` operation also deactivate the DID.
 **Note:** Operations section ensures the ITN DID Method Operations complies with [W3C DID Method Operations](https://w3c.github.io/did-core/#method-operations)
 
 
-
 ## Security Considerations
 
 For all `did:itn` DIDs, the initial asset creation, and subsequent updates are executed using `Ed25519` keys, which are widely recognized as a robust and secure cryptographic mechanism. `Confirm With Umed & Andreas - if the keys are different. DELETE THIS COMMENT LATER.`
```

Comment:
there are security guidlines in the did spec, please, list how did:itn complies with those guidelines.

Reply:
Looking into it.